.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "4.4.2"
+  - '0.10'
+  - 4.4.2
 sudo: false
+services:
+  - docker
```

Comment:
@nialldonnellyfh I had to use a docker image to run `sonar-scanner` because the sonarqube plugin for travis is broken, the new version of the runner needs a newer JDK than what is supplied by travis.

[Here's](https://travis-ci.org/feedhenry-raincatcher/raincatcher-push/jobs/162950990) a build from when I was trying to use that plugin.